app/views/editor/thang/ThangTypeColorsTabView.coffee

```diff
@@ -62,6 +63,9 @@ module.exports = class ThangTypeColorsTabView extends CocoView
       $("#shape-buttons").hide()
       $("#saved-color-tabs").show()
 
+    # Attach a stateless calculator widget
+    new ColorCalculator({ el: '#color-calculator' })
```

Comment:
Is this cleaned up when we navigate away from the page?  We may want to register a reference to it and then call `destroy()` when the view is being unloaded.  I don't know enough about Vue to know if it'll be automatically garbage collected 🤷‍♂

Reply:
Will add a destroy call. Great pickup.